### PR TITLE
Theming/Fonts + Colours files more aware of responsibilties + minor style change

### PR DESCRIPTION
**Pull request by W3stside**
Merges into #20 

1. Moves `utils` for getting colours/fonts into those respective files
2. adds `font` sizes,faces,weight to theme to use (will be used to replace current `var(--font-weight-bold)` approach we are phasing out)
3. cleans up some styles

### CODE REVIEW

File: src/theme/styles/fonts.ts [hunk 1]
```diff
@@ -24,5 +26,45 @@ const fontFace = css`
...
+}
+
+export function getFonts(mode: Theme): Fonts {
+  let FONT: Fonts
+
+  switch (mode) {
+    case Theme.LIGHT:
+    case Theme.DARK:
+      FONT = DEFAULT_FONT
+      break
```
Comment (alfetopito) [inline src/theme/styles/fonts.ts:62]: Redundant, you might as well use only `default`, or skip the switch altogether and always return `DEFAULT_FONT`

**Review by alfetopito — APPROVED** [review]
Only tiny optimization suggestion

File: src/theme/styles/fonts.ts (hunk 1 above)
Comment (W3stside) [reply src/theme/styles/fonts.ts:62]: true, true, will fixito

File: src/theme/styles/global.ts [hunk 2]
```diff
@@ -65,33 +58,26 @@ export const StaticGlobalStyle = createGlobalStyle`
...
-    text-decoration: underline;
-    cursor: pointer;
-      &:link, 
-      &:visited {
-        color: var(--color-text-active);
-      }
-  }
+  
 
+  /* TODO: move closer to H elements or sth */
```
Comment (anxolin) [inline src/theme/styles/global.ts:80]: why? imo these styles that are not themed should be sth similar to a CSS reset imo:
https://meyerweb.com/eric/tools/css/reset/

**Review by anxolin — APPROVED** [review]
Great!

**Pull request merged**